pkg/registry/query.go

```diff
@@ -30,20 +31,29 @@ func (q Querier) ListPackages(_ context.Context) ([]string, error) {
 }
 
 func (q Querier) ListBundles(_ context.Context) ([]*api.Bundle, error) {
-	var bundles []*api.Bundle
+	sliceStreamer := streamer.NewSliceStreamer()
 
+	err := q.StreamBundles(context.TODO(), sliceStreamer)
```

Comment:
Assign a name to the `ListBundles()` `context.Context` argument and use that instead of `context.TODO()`

Reply:
Addressed in the latest version of the PR.